jetty-client/src/main/java/org/eclipse/jetty/client/HttpClient.java

```diff
@@ -1109,7 +1140,7 @@ protected HttpField getAcceptEncodingField()
 
     protected String normalizeHost(String host)
     {
-        if (host != null && host.matches("\\[.*\\]"))
+        if (host != null && host.matches("\\[.*]"))
```

Comment:
Strange edit.  Is this supposed to strip out the host from an IPv6 address/host?

Reply:
Apparently, in regexp you have to escape the opening `[`, but not the closing `]`. That snippet tests whether the host name is written using the URI IPv6 notation.